src/components/App.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import axios from 'axios'
+import TodoItem from './todoItems';
+import AddTodo from './addTodo'
+import { APIHost } from '../db/settings';
+import { addNewTodo, removeTodo, toggleComplete } from '../api/helpers';
+
+export default class App extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            items: [],
+        };
+    }
+
+    componentDidMount() {
```

Comment:
Don't use these life cycle methods learn about hooks and modern react and use it.